thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/v2/AnomaliesResource.java

```diff
@@ -499,6 +501,7 @@ public void updateFeedbackForAnomalyFunctionAndTimeRange(
       @PathParam("startTime") Long startTime,
       @PathParam("endTime") Long endTime,
       @PathParam("functionId") Long functionId,
+      @QueryParam("dimensionMap") String dimensionMapJSONString,
       @QueryParam("feedbackType") String feedbackType) {
 
     if (functionId == null) {
```

Comment:
These checks can be done on with @NotNull decorator on the PathParam.

Reply:
I should have used isNotEmpty() in the first place. This is stricter than not null

Reply:
In fact changed to assign a default value to "", in case frontend don't pass in dimension information

Reply:
The constructor of dimension map doesn't take empty strings. Fixing that issue here: https://github.com/linkedin/pinot/pull/2237. Please review it and push this PR with that one.